tools/pd-ctl/pdctl/command/keyspace_command.go

```diff
@@ -87,7 +89,16 @@ func showKeyspaceNameCommandFunc(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	resp, err := doRequest(cmd, fmt.Sprintf("%s/%s?force_refresh_group_id=true", keyspacePrefix, args[0]), http.MethodGet, http.Header{})
```

Comment:
maybe we can keep `force_refresh_group_id=true` as default, but retry again when meet `groupManagerUninitializedErr`

Reply:
If no `force_refresh_group_id=true`, there will be inconsistent

Reply:
Ok, that sounds reasonable. I have change the force_refresh_group_id to default and added retry when it's groupManagerUnintalizedErr